execute_two.py

```diff
@@ -24,18 +24,18 @@ def __init__(self, menu, plots=None):
 
 DirIn,field=parser.parse(sys.argv)
 
-Vradar = Variable(None)
+
 
 
 app = QtGui.QApplication(sys.argv)
 
-MainMenu = Menu(Vradar, DirIn, name="Menu") #initiate Vradar
+MainMenu = Menu(DirIn, name="Menu") #initiate Vradar
+Vradar = MainMenu.Vradar
 
 Vtilt = Variable(0)
 Vtilt2 = Variable(0)
-Vlims = Variable(None)
-plot1 = Display(Vradar, Variable(field), Vtilt, Vlims, name="Display1", parent=MainMenu)
-plot2 = Display(Vradar, Variable(field), Vtilt2, Vlims, name="Display2", parent=MainMenu)
+plot1 = Display(Vradar, Variable(field), Vtilt, name="Display1", parent=MainMenu)
+plot2 = Display(Vradar, Variable(field), Vtilt2, name="Display2", parent=MainMenu)
 
 app.exec_()
 
```

Comment:
Hmmm, thinking as something later that we rename **execute_two.py** to **execute.py**.  This script could continue to be basic.  I think the create display would be good to have though in case the less powerful user closes a window on accident (I'm guilty of this sometimes) and wants to bring it back up.